style.css

```diff
@@ -505,6 +497,61 @@ h2.title {
   display: none;
 }
 
+.bg-modal {
+  background-color: rgba(193, 199, 208, 0.8);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 5%;
+  display: none;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-content {
+  width: 85%;
+  height: auto;
+  background-color: #fff;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin: 0 auto;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.close-button {
+  transform: rotate(45deg);
+  cursor: pointer;
+  font-size: 40px;
+  color: grey;
+  margin-top: -36px;
+}
+
+.buttons {
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+}
+
```

Comment:
- [ ] Great job here 💯 However, According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly add some space (margin) between the tags and the buttons, and try to update the header of the popup-window to match your Figma design 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/189105500-58657e97-37f5-4bad-8583-f76f56afe800.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/189105542-50862b36-173b-4e3b-a0b2-82c7d283871b.PNG" width="500" /> |